SOURCES/interface-rename-2.0.2-ExecStartPre-udevadm.XCP-ng.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/systemd/interface-rename.service b/systemd/interface-rename.service
```

Comment:
Sorry, I did not notice earlier. Please add detailed information about the reason for the patch on top, for easier future maintenance :)

Reply:
Also don't hesitate to add a changelog entry with your name, else I'll have to add mine :)

Reply:
> Sorry, I did not notice earlier. Please add detailed information about the reason for the patch on top, for easier future maintenance :)

This still holds.